app/controllers/contacts_controller.rb

```diff
@@ -0,0 +1,35 @@
+class ContactsController < ApplicationController
+
+  def index
+  end
+  
+  def create
+    @email ||= contact_params["email"]
+    @message = contact_params["message"]
+    if @email.blank?
+      flash[:error] = "Please enter an email address and try again"
+      render :index
+    else
+      flash[:success] = "Thank you for contacting us!" if mailer.deliver_now
+      redirect_to root_path
+    end
+  end
+
+  private
+
+  def mailer
+    if Setting.contact.blank?
+      contact_mailer.contact_email(@email, @message, I18n.t('mailer.default_email')) 
```

Comment:
I found it. You definitely can. And that explains why it was returning an empty string.